ivadomed/loader/adaptative.py

```diff
@@ -176,7 +176,7 @@ class BIDStoHDF5:
     """Converts a BIDS dataset to a HDF5 file.
 
     Args:
-        root_dir (str): Path to the BIDS dataset.
+        root_BIDS_list (list): List with paths to the BIDS dataset(s).
```

Comment:
why not accepting here too str and list (to avoid all this refactoring). This could be done with a decorator i guess. @ahill187 @gisellemartel can chip in

Reply:
Yah good point!

Reply:
Fixed here: https://github.com/ivadomed/ivadomed/pull/588/commits/7530f45f452000a990a841167d569bab5f527f92

Reply:
Fixed here: https://github.com/ivadomed/ivadomed/pull/588/commits/0a858ef50e35632d3e0473179121bbaded3ff81c